client/layout/index.jsx

```diff
@@ -64,7 +65,10 @@ const HELP_CENTER_STORE = HelpCenter.register();
 
 function SidebarScrollSynchronizer() {
 	const isNarrow = useBreakpoint( '<660px' );
-	const active = ! isNarrow && ! config.isEnabled( 'jetpack-cloud' ); // Jetpack cloud hasn't yet aligned with WPCOM.
+	const active =
+		! isNarrow &&
+		! config.isEnabled( 'jetpack-cloud' ) && // Jetpack cloud hasn't yet aligned with WPCOM.
+		! config.isEnabled( 'layout/dotcom-nav-redesign' ); // Dotcom nav redesign is not yet aligned with WPCOM - the handleScroll function is not yet compatible with the new layout.
```

Comment:
We might need to but I haven't found a reason to yet. I suspect we'll need to update the `handleScroll` to work with the global sidebars rather than continue with this exception so this may be removed later.

Reply:
It's breaking scroll on small screen sizes for non classic view sites, for example when I open the settings drawer. 

I guess we need to restrict this to the global pages only? Though I guess we might need scroll for some screen sizes anyway? There's quite a few things in the global site view.